### PR TITLE
Create BLorient12125

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient12125 according to the catalogue description by S. Strelcyn. I had no images available, so doubts concerning the content remain as discussed in https://github.com/BetaMasaheft/Documentation/issues/2476 and documented in a note to ms_i1.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient12125.xml [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+                                <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                                <decoNote xml:id="b4" type="Spine">Leather spine repaired in the British Library.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1800" notAfter="1978">18th or 19th century</origDate>
+                        </origin>                        
+                        <provenance>According to a note on <locus target="X"/> bought from a person with the name
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient12125.xml:181]: #

**Review by thea-m — CHANGES_REQUESTED** [review]
Very nice, thank you!

File: LondonBritishLibrary/orient/BLorient12125.xml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#229r"/>
+                                    <desc type="StampExlibris">The seal of Empress <persName ref="PRS9345taytuBe"/>.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="X"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12125.xml:159]: target always must have an `#`

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient12125.xml [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                        <provenance>According to a note on <locus target="#X"/> bought from a person with the name 
+                            <persName ref="PRS14316Alkson"/>.</provenance>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary "/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12125.xml:190]: NB: also my mistake I did not notice: the blank space in target has caused significant instability in the application, as the bibliography could not be created! https://github.com/BetaMasaheft/Manuscripts/commit/4ca7ade65d9ff291fe78380e876e4404e0898c30

Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient12125.xml:190]: this was one of the reasons behind the latest crashes

Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient12125.xml:190]: Sorry for that! I will pay more attention in the future!

**Pull request merged**